android/app/src/main/res/drawable/launch_background.xml

```diff
@@ -7,6 +7,6 @@
         <bitmap android:gravity="center" android:src="@drawable/splash"/>
     </item>
     <item>
-        <bitmap android:gravity="bottom" android:src="@drawable/branding" android:bottom="32dp"/>
```

Comment:
no need to move up a bit?

Reply:
I removed it cause it doesn't have any effects at all, and `<bitmap>`  also doesn't have attribute `android:bottom`.
RF:
- https://developer.android.com/reference/android/graphics/drawable/BitmapDrawable#summary
- https://developer.android.com/guide/topics/resources/drawable-resource#XmlBitmap

If we still need to move the branding up a bit from the bottom, the code should be:
```
<item android:bottom="32dp">
    <bitmap android:gravity="bottom" android:src="@drawable/branding"/>
</item>
```